### PR TITLE
git-gutter: Remove custom fringe bitmaps

### PR DESCRIPTION
**Pull request by real-or-random**
The config for git-gutter currently sets custom fringe bitmaps which
are optimized for half-width fringes.

I think that's an arbitrary choice. There's no spacemacs option which
enables half-width fringes and as far as I can tell, we don't optimize
for half-width fringes anywhere else. So we should optimize the
package defaults for the spacemacs default of full-size fringes.

If users set half-width fringes manually, they can customize the
bitmaps manually. Even if they don't, the bitmaps are still visible,
they just don't look as nice.

### CODE REVIEW

**Comment by real-or-random** [comment]
Screenshots with full-width fringes (the default)

Before: Screenshot from 2021-12-17 10-39-07, After: Screenshot from 2021-12-17 10-37-20

The symbols on the left are hardly visible, I think.

**Comment by lebensterben** [comment]
I've no idea why this is first added in https://github.com/syl20bnr/spacemacs/pull/3632

It was quite too long ago...

I've no issue with this PR but let's wait for approvals from other maintainers.

**Review by lebensterben — APPROVED** [review]

**Review by smile13241324 — APPROVED** [review]
No idea why custom fringes have been added but when the original ones work well why keep the exception. So lets have this merged and see how it plays out.

**Pull request merged**
